.storybook/setupTwig.js

```diff
@@ -0,0 +1,26 @@
+import { resolve } from 'path';
+import twigDrupal from 'twig-drupal-filters';
+import twigBEM from 'bem-twig-extension';
+import twigAddAttributes from 'add-attributes-twig-extension';
+
+export const namespaces = {
```

Comment:
Did this section need to be added here? We handle these in `./.storybook/webpack.config.js`

Reply:
We do need to export this from a centralized place so that I can pull the namespaces into the test twig rendering stuff. However, I think I can consolidate this, and use it in both the webpack config _and_ the test config. Does that sound like a good path forward?

Reply:
Yea, that sounds great - having these in one location would be awesome